kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/KieServerConstants.java

```diff
@@ -30,6 +30,7 @@
     public static final String KIE_OPTAPLANNER_SERVER_EXT_DISABLED = "org.optaplanner.server.ext.disabled";
     public static final String KIE_CASE_SERVER_EXT_DISABLED = "org.jbpm.case.server.ext.disabled";
     public static final String KIE_DMN_SERVER_EXT_DISABLED = "org.kie.dmn.server.ext.disabled";
+    public static final String KIE_JBM_QUERIES_SERVER_EXT_DISABLED = "org.jbpm.taskqueries.server.ext.disabled";
```

Comment:
shouldn't the value of the property not include task any more as it's not only for tasks?